src/app/file-manager/models/item-status.ts

```diff
@@ -4,4 +4,5 @@ export class ItemStatus {
   static readonly UPLOADING = 'Uploading';
   static readonly COMPLETED = 'Completed';
   static readonly CANCELED = 'Canceled';
+  static readonly INDETERMINATE = 'indeterminate';
```

Comment:
This appears to be unused

Reply:
Yes, it was a mix of concepts  so it was moved to another property. Will remove it.